src/interface/src/app/features/features.json

```diff
@@ -1,7 +1,8 @@
 {
   "login": true,
-  "unlaunched_layers": true,
-  "top_percent_slider":true,
+  "show_socal": false,
```

Comment:
Do we want to explicitly set this in our repo? (Not a bit deal, but may cause some confusion later on since the other flag values are defaulted to true until they're set locally)

Reply:
good point.  I'll turn those off too.